Bottom: e07eb7ce8a0b1bf8bc4b3b5cb99b1ddf5dca46a1
Top:    5bd97bdf8483863e3c8a4c7080b6fc9d81a05191
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 22:13:53 +0100

Refresh of hid-gtk-remove-the-unused-func

---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 7136fbb..0fcb5d2 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -838,34 +838,3 @@ ghid_scrolled_text_view (GtkWidget * box,
     *scr = scrolled;
   return view;
 }
-
-
-/* If src is not utf8, *dst is converted to utf8.
- */
-gboolean
-utf8_dup_string (gchar ** dst_utf8, const gchar * src)
-{
-  if (!dst_utf8 || (!*dst_utf8 && !src))
-    return FALSE;
-  if (*dst_utf8)
-    {
-      if (src && !strcmp (*dst_utf8, src))
-	return FALSE;
-      g_free (*dst_utf8);
-    }
-  if (src)
-    {
-      if (g_utf8_validate (src, -1, NULL))
-	*dst_utf8 = g_strdup (src);
-      else
-	{
-	  *dst_utf8 = g_locale_to_utf8 (src, -1, NULL, NULL, NULL);
-	  if (!*dst_utf8)
-	    *dst_utf8 = g_strdup (src);
-	}
-    }
-  else
-    *dst_utf8 = NULL;
-
-  return TRUE;
-}
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 12f9bc2..5e72ba2 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -353,7 +353,6 @@ void make_route_style_buttons (GHidRouteStyleSelector *rss);
 /* gui-utils.c
 */
 gboolean dup_string (gchar ** dst, const gchar * src);
-gboolean utf8_dup_string (gchar ** dst_utf8, const gchar * src);
 void free_glist_and_data (GList ** list_head);
 
 ModifierKeysState ghid_modifier_keys_state (GdkModifierType * state);
